accessibility-oscilloscope/overlay/usr/bin/pipe-test

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/python3 -u
+import sys
+import re
+
+re_y = re.compile(r'BTN_[0-3]\s+(\d+)')
+
+if __name__ == "__main__":
+    for line in sys.stdin:
+        reg = re_y.search(str(line))
+        if reg:
+            print(reg.group(1))
```

Comment:
would like to figure out why the `awk` pipeline this is replacing failed before merging -- considering it's being used for both `wacom` scripts and seems to be working fine for the other.

the python program this is based on (`monstrosity`) was a stopgap i was also hoping to replace, just based on the like ~5s startup time of the python interpreter.